tests/lost_reset/test

```diff
@@ -26,34 +26,32 @@ system("auditctl -D >& /dev/null");
 my $result;
 my $i;
 for ($i = 0; $i < $iterations; $i++) {
-# Kill the daemon, set the buffers low, set the wait time to 1ms, turn on auditing
-system("service auditd stop >/dev/null 2>&1");
-system("auditctl -D >/dev/null 2>&1");
-system("auditctl -b 1 >/dev/null 2>&1");
-system("auditctl --backlog_wait_time 1 >/dev/null 2>&1");
-system("auditctl -e 1 >/dev/null 2>&1");
-
-###
-# tests
-# Start floodping to generate activity
-seek($fh_tmp, 0, 0);
-system("ping -f 127.0.0.1 >/dev/null 2>&1 & echo \$! >$tmpout");
-my $ping_pid = <$fh_tmp>;
-chomp($ping_pid);
-# Add rule to generate audit queue events from floodping
-$result = system("auditctl -a exit,always -S all -F pid=$ping_pid >/dev/null 2>&1");
-ok($result, 0); # Was the rule added successfully?
-sleep 1;
-kill 'TERM', $ping_pid;
-system("auditctl -d exit,always -S all -F pid=$ping_pid >/dev/null 2>&1");
-
-# Restart the daemon to collect messages in the log
-system("service auditd start >/dev/null 2>&1");
-sleep 1;
+	# Kill the daemon, set the buffers low, set the wait time to 1ms, turn on auditing
+	system("service auditd stop >/dev/null 2>&1");
+	system("auditctl -D >/dev/null 2>&1");
+	system("auditctl -b 1 >/dev/null 2>&1");
+	system("auditctl --backlog_wait_time 1 >/dev/null 2>&1");
+	system("auditctl -e 1 >/dev/null 2>&1");
+	
```

Comment:
I didn't think this needed to be said, but please don't add a tab/indent to blank lines.